tests/validate_must_gather_test.go

```diff
@@ -120,7 +122,9 @@ var _ = Describe("validate the must-gather output", func() {
 				crDir := path.Join(crsDir, cr.Name())
 				crFiles, err := os.ReadDir(crDir)
 				Expect(err).ToNot(HaveOccurred())
-				Expect(crFiles).Should(HaveLen(1))
+				if !strings.Contains(cr.Name(), "instancetype") {
+					Expect(crFiles).Should(HaveLen(1))
+				}
```

Comment:
is this needed just because https://github.com/openshift/release/pull/42972 is not merged yet, and test cluster version is ocp/4.11?

Reply:
would it make sense to change `expectedResources := map[string]bool` to `expectedResources := map[string]int` and have the expected number of resources there?

which would be the vmci and vmcp counts, and 1 for the others

Reply:
`Expect(crFiles).Should(HaveLen(expectedResources[cr.Name()]))`

Reply:
I'm not sure. If common instance types are added/deprecated, maybe this number can get changed. Not sure if it would be wise to hard code it. What if I'll change it to: 
`Expect(len(crFiles)).Should(BeNumerically(">=", 1))` without skipping vmci and vmcp? 
This way we verify they are not empty and don't skip them.

Reply:
but this cluster is always the same, fresh for each test, the idea would to make the tests more simple

wdyt @nunnatsa ?

Reply:
@nunnatsa WDYT?